cdap-metadata-spi/pom.xml

```diff
@@ -36,6 +36,12 @@
       <artifactId>cdap-api</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
what is this needed for?

Reply:
asking because we tried to keep the SPI dependencies to a minimum, ideally only depending on API.

Reply:
oh it is only for tests, that is fine then.

Reply:
Yep, added to enable Tasks.waitFor() in async tests within MetadataStorageTests.